packages/showcases/css/stories/components/select/examples/disabled.html

```diff
@@ -0,0 +1,17 @@
+<div class="block">
+  <div class="vtmn-select">
+    <div class="vtmn-select_input">
+      <select name="sports" id="my-select" autocomplete disabled>
+        <option value="">Select</option>
+        <option value="gymnastics">Gymnastics</option>
+        <option value="archery">Archery</option>
+        <option value="climbing">Climbing</option>
+        <option value="surf">Surf</option>
+        <option value="badminton">Badminton</option>
+        <option value="football">Football</option>
+      </select>
+
+      <label for="my-select">Label</label>
```

Comment:
Please don't reverse `label` and `select` HTML tags. Plus, we don't need an extra `div` incapsulated.